content/soarian/r4/foundation/documents/document-reference.md

```diff
@@ -0,0 +1,151 @@
+* [Document category](https://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.category){:target="_blank"}
+* [Subject (Patient)](https://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.subject){:target="_blank"}
+* [Created Date](https://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.date){:target="_blank"}
+* [Author](https://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.author){:target="_blank"}
+* [Custodian](https://hl7.org/fhir/R4/documentreference-definitions.html#DocumentReference.custodian){:target="_blank"}
+* [Security Label](https://hl7.org/fhir/R4/documentreference-definitions.html#DocumentReference.securityLabel){:target="_blank"}
+* [Content](https://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.content){:target="_blank"}
+  * [Attachment](https://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.content.attachment){:target="_blank"}
+      * [Attachment ContentType](https://hl7.org/fhir/r4/datatypes-definitions.html#Attachment.contentType){:target="_blank"}
+      * [Attachment Data](https://hl7.org/fhir/R4/datatypes-definitions.html#Attachment.data){:target="_blank"}
+      * [Attachment URL](https://hl7.org/fhir/r4/datatypes-definitions.html#Attachment.url){:target="_blank"}
+  * [Format](https://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.content.format){:target="_blank"}
+* [Patient encounter]( https://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.context.encounter){:target="_blank"}
+* [Document period]( https://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.context.period){:target="_blank"}
+
+
+## Terminology Bindings
+
+<%= terminology_table(:soarian_document_reference, :r4) %>
+
+## Search
+
+Search for DocumentReference resources that meet the specified query parameters:
+
+    GET /DocumentReference?:parameters
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true) %>
+
+### Parameters
+
+ Name                     | Required?          | Type          | Description
+--------------------------|--------------------|---------------|--------------------------------------------------------------------------------------------------------
+ `_id`                    | This, or `patient` | [`token`]     | The logical resource ID associated with the resource.
+ `patient`                | This, or `_id`     | [`reference`] | The patient who has the document referenced. 
+`category`                | No                 | [`token`]     | The categorization for the type of document referenced. Example: `category=clinical-note`
+`type`                    | No                 | [`token`]     | The kind of document referenced. Example: `https://loinc.org/11488-4`
+`date`                    | No                 | [`date`]      | Date range into which the document referenced falls. Example: `date=ge2020-01-01`
```

Comment:
should it be date_time ?

Reply:
We are keeping only date in order to make it consistent with the other resources.